app/controllers/courses_controller.rb

```diff
@@ -426,13 +426,13 @@ def edit_course_params
 
   def categorize_courses_for_listing(courses)
     listing = {}
-    listing[:disabled] = []
 
     # temporal
     listing[:current] = []
     listing[:completed] = []
     listing[:upcoming] = []
 
+	listing[:disabled] = []
```

Comment:
A tab was used here although the rest of the lines uses spacing so there's a spacing difference

Reply:
resolved!